worldwind/src/commonMain/kotlin/earth/worldwind/shape/Path.kt

```diff
@@ -148,9 +159,9 @@ open class Path @JvmOverloads constructor(
 
         // Configure the drawable to display the shape's extruded interior.
         if (activeAttributes.isDrawInterior && isExtrude) {
-            drawState.color(if (rc.isPickMode) pickColor else activeAttributes.interiorColor)
-            drawState.opacity(if (rc.isPickMode) 1f else rc.currentLayer.opacity)
-            drawState.drawElements(
+            extrudeDrawState.color(if (rc.isPickMode) pickColor else activeAttributes.interiorColor)
```

Comment:
Can pickColor can be pre stored in attributes?

Reply:
solved

Reply:
solved